Scripts/Python/xYeeshaPages.py

```diff
@@ -72,40 +72,11 @@
 kReltoLinkSecs = 4.0
 
 #Note: when adding more Ages to this script, be sure to also update IDrawLinkPanel() module below
-kYeeshaPage01 = 201
-kYeeshaPage02 = 202
-kYeeshaPage03 = 203
-kYeeshaPage04 = 204
-kYeeshaPage05 = 205
-kYeeshaPage06 = 206
-kYeeshaPage07 = 207
-kYeeshaPage08 = 208
-kYeeshaPage09 = 209
-kYeeshaPage10 = 210
-kYeeshaPage12 = 212
-kYeeshaPage13 = 213
-kYeeshaPage14 = 214
-kYeeshaPage15 = 215
-kYeeshaPage16 = 216
-kYeeshaPage17 = 217
-kYeeshaPage18 = 218
-kYeeshaPage19 = 219
-kYeeshaPage20 = 220
-kYeeshaPage21 = 221
-kYeeshaPage22 = 222
-kYeeshaPage23 = 223
-kYeeshaPage24 = 224
-kYeeshaPage25 = 225
-kYeeshaPage26 = 226
+kYeeshaPage = []
+for x in range(0,99):
+    kYeeshaPage.insert(x,200+x)
```

Comment:
```suggestion
kYeeshaPage = list(range(200, 298, 1))
```

Stopping at 298 due to 299 colliding with `kYeeshaPageCancel`.